src/compiler/scanner.ts

```diff
@@ -283,12 +283,12 @@ namespace ts {
     /**
      * Test for whether a single line comment's text contains a directive.

```

Comment:
Consider updating the comment to reflect the fact that leading whitespace is expected to have been trimmed.